src/style.css

```diff
@@ -1 +1,93 @@
 /* empty source */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./stars.css";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;

```

Comment:
Don't use `*` for margin and padding. It's not a good practice. Leave only box-sizing here